pravega/pravega-tools
# Issue 43 password file creation command

**Change log description**
PasswordFileCreation tool to generate a file based user database for PasswordAuthHandler

**Purpose of the change**
Fixes #43 

**What the code does**

Creates a command that takes as argument the name of the file to be generated and user details (username:password:acl). Encrypted password is generated from the user details. It then creates a file with filename given as argument and writes the encrypted password to the file. 

Registering the command in the ImmutableMap 'builder'.

Test file for the command

How to verify it
List the commands available in Pravega CLI. Use the new command and check its functionality.
